entrypoint.sh

```diff
@@ -12,4 +12,6 @@ if [ ! -f "$snap" ]; then
   try-runtime --runtime existing create-snapshot --uri $url $snap
 fi
 
-try-runtime --runtime $runtime on-runtime-upgrade --checks=$checks $* snap --path $snap
+if [ -f "$runtime" ];then
+  try-runtime --runtime $runtime on-runtime-upgrade --checks=$checks $* snap --path $snap
+fi
```

Comment:
I am not sure this will do what you want since if `runtime == SKIP` this will still evaluate to true?

Reply:
check `man [`

then scroll down to;

```
       -f FILE
              FILE exists and is a regular file
```

Any non existing argument will skip